src/WpfKata/MainViewModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace WpfKata
+{
+    public sealed class MainViewModel : INotifyPropertyChanged
+    {
+        private string _firstName;
+        private string _lastName;
+        private int _age;
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public string FirstName
```

Comment:
Didn't the INPC analyzer pick up on if you placed the field directly above the property?

Reply:
It works; I was moving too fast and overlooked it!